packages/lit-analyzer/package.json

```diff
@@ -46,7 +46,7 @@
 		"didyoumean2": "4.1.0",
 		"fast-glob": "^2.2.6",
 		"parse5": "5.1.0",
-		"ts-simple-type": "~1.0.5",
+		"ts-simple-type": "2.0.0-next.0",
```

Comment:
I assume this is intentionally an `=` equivalent instead of `^` because pre-releases are unstable?

Reply:
Also why did we need the bump?

Reply:
I was trying to see if a newer ts-simple-type would fix the error I was seeing. This should have been bumped anyway.

Reply:
I'm going to back this out and see if it fixes the build. For some reason `npm run bootstrap` isn't regenerating package locks and I think that's causing `npm ls` to fail and I don't want to deal with that now.